ansible_collections/arista/cvp/molecule/cv_tag_v3/test_cv_tag_v3_device.yml

```diff
@@ -0,0 +1,232 @@
+        tags: "{{ASSIGN_UNASSIGN_DELETE_EXISTING_TAGS}}"
+        mode: delete
+      register: TAGS_DELETE
+
+    - name: Check delete device tags
+      assert:
+        that:
+          - TAGS_DELETE.changed == true
+          - TAGS_DELETE.failed == false
+          - TAGS_DELETE.success == true
+          - TAGS_DELETE.tags_manager.tags_manager_list != []
+          - TAGS_DELETE.tags_manager.tags_manager_count == 1
+
+# Create -> assign -> delete
+
+    - name: Create device tags (part of CREATE -> assign -> delete)
+      arista.cvp.cv_tag_v3:
+        tags: "{{CREATE_TAGS}}"
+        mode: create
+      register: TAGS_CREATE
+
+    - name: Check create device tags
+      assert:
+        that:
+          - TAGS_CREATE.changed == true
+          - TAGS_CREATE.failed == false
+          - TAGS_CREATE.success == true
+          - TAGS_CREATE.tags_manager.tags_manager_list != []
+          - TAGS_CREATE.tags_manager.tags_manager_count == 1
+
+    - name: Assign existing device tags (part of create -> ASSIGN -> delete)
+      arista.cvp.cv_tag_v3:
+        tags: "{{ASSIGN_UNASSIGN_DELETE_EXISTING_TAGS}}"
+        mode: assign
+      register: ASSIGN_EXISTING
+
+    - name: Check assign existing device tags
+      assert:
+        that:
+          - ASSIGN_EXISTING.changed == true
+          - ASSIGN_EXISTING.failed == false
+          - ASSIGN_EXISTING.success == true
+          - ASSIGN_EXISTING.tags_manager.tags_manager_list != []
+          - ASSIGN_EXISTING.tags_manager.tags_manager_count == 1
+
+    - name: Delete device tags (part of create -> assign -> DELETE)
+      arista.cvp.cv_tag_v3:
+        tags: "{{ASSIGN_UNASSIGN_DELETE_EXISTING_TAGS}}"
+        mode: delete
+      register: TAGS_DELETE
+
+    - name: Check delete device tags
+      assert:
+        that:
+          - TAGS_DELETE.changed == true
+          - TAGS_DELETE.failed == false
+          - TAGS_DELETE.success == true
+          - TAGS_DELETE.tags_manager.tags_manager_list != []
+          - TAGS_DELETE.tags_manager.tags_manager_count == 1
+
+# Negative Test
+# Commenting negative testcases since its failing currently
+# Tags on auto_create: false gets created on System Tags
+
+    # - name: Negative Test assign non-existing device
+    #   arista.cvp.cv_tag_v3:
+    #     tags: "{{ASSIGN_NON_EXISTING_TAGS}}"
+    #     mode: assign
+    #     auto_create: false
+    #   register: ASSIGN_NON_EXISTING
+    #   ignore_errors: yes
+
+    # - name: Check assign non-existing device tags
+    #   assert:
+    #     that:
+    #       - ASSIGN_NON_EXISTING.changed == false
+    #       - ASSIGN_NON_EXISTING.failed == true
+    #       - ASSIGN_NON_EXISTING.success == false
+    #       - ASSIGN_NON_EXISTING.tags_manager.tags_manager_list == []
+    #       - ASSIGN_NON_EXISTING.tags_manager.tags_manager_count == 0
+
+    # - name: Negative Test delete non-existing device tags
+    #   arista.cvp.cv_tag_v3:
+    #     tags: "{{ASSIGN_DELETE_NON_EXISTING_TAGS}}"
```

Comment:
You need `device` keyword here to make this work!

Reply:
Yes, giving it in the input.